File: client/src/app/_dtos/hr/audioMessageDto.ts
export interface IAudioMessageDto
{
    checked: boolean;
    id: number;
    applicationNo: number;
    candidateName: string;
    messageText: string;
    subject: string;
    dateComposed: Date;
    datePlayedback: Date;
    fileName: string;
    feedbackReceived: number;
}

File: api/DTOs/HR/AudioMessageDto.cs
namespace api.HR.DTOs
{
    public class AudioMessageDto
    {
        public int Id { get; set; }
        public string Subject { get; set; }
        public string CandidateName { get; set; }
        public int ApplicationNo { get; set; }
        public string MessageText { get; set; }
        public DateTime DateComposed { get; set; }
        public DateTime DatePlayedback { get; set; }
        public string FileName { get; set; }
        public int FeedbackReceived { get; set; }=0;
    }
}